usaspending_api/common/filters/time_period/decorators/new_awards_only.py

```diff
@@ -0,0 +1,87 @@
+from usaspending_api.common.filters.time_period.time_period import ITimePeriod
+from usaspending_api.search.filters.elasticsearch.filter import _QueryType
+
+NEW_AWARDS_ONLY_KEYWORD = "new_awards_only"
+
+
+class NewAwardsOnlyTimePeriod(ITimePeriod):
+    """A decorator class that can be used to apply a new awards only filter
```

Comment:
Since "Decorators" in python have their own connotation (like attributes in .net and annotations in java), probably change this to say
```
"""A composable class that can be used according to the Decorator software design pattern, to apply ...
```

Reply:
Ok.